### PR TITLE
Dev/new games

### PR DESCRIPTION
**Pull request by deadshot465**
WIP.

### CODE REVIEW

File: BurstBotNET/Commands/Start.cs [hunk 1]
```diff
@@ -1,55 +1,91 @@
...
-                .WithDescription(
-                    "Congratulations! You can now start joining games to play poker with other people!\nAs a first time bonus, **you also have received 100 tips!**\nEnjoy!")
-                .WithThumbnail(botUser.GetAvatarUrl(ImageFormat.Auto))
-                .WithTitle("Welcome to the All Burst!")
-                .AddField("Current Tips", (await playerData.GetJsonAsync<RawPlayer>()).Amount.ToString())));
+        var embed = new Embed(
+            Author: new EmbedAuthor(member.GetDisplayName(), IconUrl: _context.User.GetAvatarUrl()),
+            Colour: BurstColor.Burst.ToColor(),
+            Description:
+            "Congratulations! You can now start joining games to play poker with other people!\nAs a first time bonus, **you also have received 100 tips!**\nEnjoy!",
```
Comment (MorphTuple) [inline BurstBotNET/Commands/Start.cs:75]: Localization?

File: BurstBotNET/Commands/Start.cs [hunk 2]
```diff
@@ -1,55 +1,91 @@
...
+            Thumbnail: new EmbedThumbnail(botUser.GetAvatarUrl()),
+            Title: "Welcome to the Jack of All Trades!",
+            Fields: new[]
+            {
+                new EmbedField("Current Tips", (await playerData.GetJsonAsync<RawPlayer>()).Amount.ToString())
```
Comment (MorphTuple) [inline BurstBotNET/Commands/Start.cs:80]: Localization?

File: BurstBotShared/Shared/ErrorMessages.cs [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+{
+    public const string InvalidPlayer =
+        "Sorry, but either one of the players you invited hasn't joined the server yet, or he doesn't have enough tips to join a game!";
+
+    public const string HandleReactionFailed =
+        "Failed to handle reaction from invited players";
+
+    public const string PlayerStateNoGameProgress = "Player state doesn't have game progress.";
+
+    public const string JoinNotInGuild = "Sorry, but you can only join games in a guild!";
```
Comment (MorphTuple) [inline BurstBotShared/Shared/ErrorMessages.cs:13]: Localization?

**Review by MorphTuple — CHANGES_REQUESTED** [review]
This is a massive PR. I can't really put a detailed review however looking through at it there's nothing that really stands out as unacceptable. However I think the codebase for now is half localized? That's probably my only issue for now. Everything else LGTM.

**Comment by deadshot465** [comment]
> This is a massive PR. I can't really put a detailed review however looking through at it there's nothing that really stands out as unacceptable. However I think the codebase for now is half localized? That's probably my only issue for now. Everything else LGTM.

That's a really good question because I'm hesitant if I want to localize the whole bot. Right now only games are localized and even they are not fully localized. Could you please check if there are other issues and just leave localization alone for now? Because I don't really want to deal with the decision about localization now XD

**Comment by MorphTuple** [comment]
I think I would need to actually pull and run it locally and explore the code that way. Please remind me again in a bit 😅

**Pull request merged**